jetbrains-gateway/README.md

```diff
@@ -41,18 +41,52 @@ module "jetbrains_gateway" {
 }
 ```
 
-### Use the latest release version
+### Use fixed versions set by `jetbrains_ide_versions`
 
 ```tf
 module "jetbrains_gateway" {
   source         = "registry.coder.com/modules/jetbrains-gateway/coder"
-  version        = "1.0.23"
+  version        = "1.0.25"
   agent_id       = coder_agent.example.id
   agent_name     = "example"
   folder         = "/home/coder/example"
   jetbrains_ides = ["GO", "WS"]
   default        = "GO"
-  latest         = true
+  latest         = false
+  jetbrains_ide_versions = {
```

Comment:
Do they have to define all of them?
As an operator I'd probably just care about the one(s) my team uses.

Reply:
No they don't have to. They can shorten this to the ones they use.

Reply:
Let's keep the suggestions realistic then; let's maybe specify IntelliJ+PyCharm and add a comment saying that one or more can be defined?

Reply:
Will do.

Reply:
updated.